web/cobrands/sass/_layout.scss

```diff
@@ -252,15 +271,47 @@ body.mappage.admin {
   height: auto; // stretch from bottom of header to bottom of window
   margin: 0;
 
+  @media (min-width: $screen-size--medium) {
+    #{$left}: $mappage-sidebar-width--medium;
+  }
+
+  @media (min-width: $screen-size--large) {
+    #{$left}: $mappage-sidebar-width--large;
+  }
+
+  @media (min-width: $screen-size--xlarge) {
+    #{$left}: $mappage-sidebar-width--xlarge;
+  }
+
   .with-notes & {
     #{$left}: $mappage-sidebar-width + $mappage-notes-width;
+
+    @media (min-width: $screen-size--medium) {
+      #{$left}: $mappage-sidebar-width--medium + $mappage-notes-width--medium;
+    }
+
+    @media (min-width: $screen-size--large) {
+      #{$left}: $mappage-sidebar-width--large + $mappage-notes-width--large;
+    }
+
+    @media (min-width: $screen-size--xlarge) {
+      #{$left}: $mappage-sidebar-width--xlarge + $mappage-notes-width--xlarge;
+    }
   }
 
   .with-actions & {
     #{$left}: $mappage-actions-sidebar-width + $mappage-actions-width;
 
-    @media (max-width: 79em) {
-      #{$left}: $mappage-sidebar-width--medium  + $mappage-actions-width--medium;
+    @media (min-width: $screen-size--medium) {
+      #{$left}: $mappage-sidebar-width--medium + $mappage-actions-width--medium;
```

Comment:
The variable names were/are very confusing here, sorry.

With `.with-actions`, the map_sidebar width was either mappage-actions-sidebar-width + mappage-actions-width, or at <79em **mappage-sidebar-width--medium** + mappage-actions-width--medium. Which I know you've kept here, but you've now also started using that for the general width of the sidebar _without_ notes/actions as well, when it was only previously for with actions. If you did an equivalent to your second video when logged out, I think you'd see the sidebar could then be wider throughout (because there's no blue column), we don't need it to be the same for both.

Your variable names in this PR are better now, as you're varying the sidebar width in general, but I think you need to use your new mappage-actions-sidebar-width variables here and in the equivalent other place rather than what you have here. That then lets you specify different widths for when the actions bar is there and when it isn't.

Let me know if that doesn't make sense!